README.md

```diff
@@ -25,8 +25,16 @@ Standards RFCs shouldn’t be substantially altered after they are accepted, alt
 3. When the new RFC is accepted and the PR is merged, the old RFC will no longer be active.
 
 ## Managing Action Plans
+
 For RFCs where the outcome is an agreed Action Plan, you may want to update the RFC with meaningful status updates in new PRs. Once the plan is either complete or no longer relevant, it should be moved to the archived directory in a new PR.
 
+## Raising a concern with an RFC
+
+There is no definitive process for discussing an RFC before the above process has been started, or after it has finished. To faciliate this, an [Issue][Issues] may be created to discuss potential changes.
```

Comment:
My understanding is that new RFCs can be opened "Request for Comment" that can obsolete previous RFCs to update the wording to match current behaviour

Reply:
Unless we open blank PRs, the reason for suggesting Issues was to gauge opinion on a change to an RFC, or even a new RFC, before investing the time in writing one.